config/PocketCasts.prototype.xcconfig

```diff
@@ -0,0 +1,12 @@
+#include "PocketCasts.base.xcconfig"
+
+// Redefining doesn't seem to work, Xcode resolves it as ".prototype" only
+//PRODUCT_BUNDLE_IDENTIFIER_ROOT = $(PRODUCT_BUNDLE_IDENTIFIER_ROOT).prototype
+PRODUCT_BUNDLE_IDENTIFIER_ROOT = au.com.shiftyjelly.podcasts.prototype
+PRODUCT_BUNDLE_IDENTIFIER = $(PRODUCT_BUNDLE_IDENTIFIER_ROOT)
+
+DEVELOPMENT_TEAM = 99KV9Z6BKV // Automattic, Inc. (Enterprise)
+
+// Notice that because currently all extensions use this xcconfig for the Prototype bulid configuration,
+// this line is enough to set the Provisioning Profiles for all.
+PROVISIONING_PROFILE_SPECIFIER = match InHouse $(PRODUCT_BUNDLE_IDENTIFIER)
```

Comment:
Should the prototype build run with the `-D DEBUG` flag too?

I guess it depends on the kind of behavior you expect to test with a prototype build... At the same time, I wouldn't recommend basing much logic on preprocessor macro checks, so maybe it's okay as it is?

```suggestion
PROVISIONING_PROFILE_SPECIFIER = match InHouse $(PRODUCT_BUNDLE_IDENTIFIER)

OTHER_SWIFT_FLAGS = -D DEBUG
```

Likewise, we could also make it point to the staging backend

```suggestion
PROVISIONING_PROFILE_SPECIFIER = match InHouse $(PRODUCT_BUNDLE_IDENTIFIER)

OTHER_SWIFT_FLAGS = -D STAGING -D DEBUG
```

Reply:
I guess `DEBUG` is fine. We have the additional developer settings (the place where we will add the Prod/Staging toggle) that will also be available so there is no need to add the `STAGING` flag.